### DISCUSSION TITLE
Can we add something like this to versioning?

### DISCUSSION THREAD

**Discussion by berkantsahn** [mkdocs/mkdocs#3174] [Ideas]
A suggestion came to my mind for the versioning process. Maybe we can do something like this already, but I couldn't find an example.
So, for example, we have versions 1.0.0 and 2.0.0 in our project. 

When I add a new file to my project in version 2.0.0, is there any chance of it being "new" icon next to it? Or can something like this be done? Or is it possible for me to have something like "update" icon next to it when I change a file?

**Comment by Andre601** [comment]
If you're using Material for MkDocs and have Insider access could you use the "Navigation Status" feature it offers:
https://squidfunk.github.io/mkdocs-material/reference/#setting-the-page-status

If not then, the only way I can imagine would be to extend whatever theme you use, override the right file and insert some function, like checking if the file has specific metadata present to display icons or whatever...

**Reply by berkantsahn** [reply]
Thanks. I am appreciate.

**Comment by berkantsahn** [comment]
You can follow the link for result: https://github.com/squidfunk/mkdocs-material/discussions/5338
